### PR TITLE
build/configs/rtl8721csm/loadable_apps: Update defconfig with .config

### PR DESCRIPTION
**Pull request by sangam-swami**
Update defconfig, fix build break with loadable config

### CODE REVIEW

File: build/configs/rtl8721csm/loadable_apps/defconfig [hunk 1]
```diff
@@ -550,6 +558,7 @@ CONFIG_UART2_2STOP=0
 # Wireless Device Options
 #
 CONFIG_DRIVERS_WIRELESS=y
+CONFIG_VIRTUAL_WLAN=y
```
Comment (jeongchanKim) [inline build/configs/rtl8721csm/loadable_apps/defconfig:561]: @pillip8282 , Could you check we can enable this by default?

File: os/drivers/wireless/virtual/vdev.c [hunk 2]
```diff
@@ -303,12 +303,6 @@ void vwifi_start(void)
 	return;
 }
 
-void up_netinitialize(void)
```
Comment (jeongchanKim) [inline os/drivers/wireless/virtual/vdev.c:306]: Is it okay to remove this function?
As I know, this function is required for every board/driver which uses from `up_initialize()`.

Comment (sangam-swami) [reply os/drivers/wireless/virtual/vdev.c:306]: the definition is already present at amebad_enet.c, so it is giving conflicts

**Review by thapav — APPROVED** [review]
LGTM

**Review by abhishek-samsung — APPROVED** [review]
LGTM

**Comment by dineshdas-samsung** [comment]
LGTM

File: build/configs/rtl8721csm/loadable_apps/defconfig [hunk 3]
```diff
@@ -1384,8 +1409,8 @@ CONFIG_ENABLE_ENV_GET=y
 CONFIG_ENABLE_ENV_SET=y
 CONFIG_ENABLE_ENV_UNSET=y
 CONFIG_ENABLE_FREE=y
-# CONFIG_ENABLE_IRQINFO is not set
 CONFIG_ENABLE_KILL=y
+CONFIG_ENABLE_KILLALL=y
```
Comment (deepaksrma) [inline build/configs/rtl8721csm/loadable_apps/defconfig:1413]: can we keep it disable?
because kill command is already enabled.

Comment (jeongchanKim) [reply build/configs/rtl8721csm/loadable_apps/defconfig:1413]: `kill` and `killall` have different functionality, so I think we can enable both.

File: os/drivers/wireless/virtual/vdev.c (hunk 2 above)
Comment (sunghan-chang) [reply os/drivers/wireless/virtual/vdev.c:306]: @pillip8282 Could you review this deletion?

**Comment by sunghan-chang** [comment]
@sangam-swami Could you resolve conflicts?

File: os/drivers/wireless/virtual/vdev.c (hunk 2 above)
Comment (pillip8282) [reply os/drivers/wireless/virtual/vdev.c:306]: if you disable virtual driver option then this file isn't built. You can't remove this function

File: build/configs/rtl8721csm/loadable_apps/defconfig (hunk 1 above)
Comment (pillip8282) [reply build/configs/rtl8721csm/loadable_apps/defconfig:561]: No it's not default. virtual_wlan is mocking driver to test wi-fi manager. So it have to be disabled in here.

File: os/drivers/wireless/virtual/vdev.c (hunk 2 above)
Comment (jeongchanKim) [reply os/drivers/wireless/virtual/vdev.c:306]: As @pillip8282  said, we should not remove this. Please revert it.

**Comment by sunghan-chang** [comment]
@sangam-swami Please check Jeongchan and my comments.

**Comment by sangam-swami** [comment]
This issue is fixed by commit : 19db4198dc63d937bfc55ee54892b6de6a91ba94
so closing the PR

**Pull request closed without merging**